mlrun/datastore/targets.py

```diff
@@ -390,6 +390,7 @@ class BaseStoreTarget(DataTargetBase):
     is_offline = False
     support_spark = False
     support_storey = False
+    support_pandas = False
```

Comment:
Looks like some targets that support pandas are missing `support_pandas = True`. E.g. `TSDBTarget`.

Reply:
I am trying to understand, so why if got `NotImplementedError` on `as_df` function?

Reply:
Actually you're right. Please add `support_pandas = ...` to all targets, to be consistent.